Assignment 1(question 2)

Suruchibharti opened the pull request:
"Submitted Assignment-1(Question-2) on 6th october"

Suruchibharti committed:
Create CRUD_Operations.c

CRUD OPERATIONS  perform on file .

Suruchibharti committed:
Update CRUD_Operations.c

added commetns for better understanding of the code

arti-niraj-yadav12 reviewed (commented):
Did you execute this script code?
Is It running properly?

Suruchibharti commented:
> Did you execute this script code? Is It running properly?

Yes, I have executed the script. It’s running properly.

arti-niraj-yadav12 commented:
But when I ran your code It's throwing an error

Suruchibharti committed:
Update CRUD_Operations.c

header file added

Suruchibharti committed:
Update CRUD_Operations.c

arti-niraj-yadav12 commented on CRUD_Operations.c:
```diff
@@ -0,0 +1,156 @@
+#define FILE_NAME "records.txt"
+
+struct Record {
+    int id;
```
give meaningful name to all variables
id -> user_id

arti-niraj-yadav12 commented on CRUD_Operations.c:
```diff
@@ -0,0 +1,156 @@
+//if fille not exist then create file: records.txt
+void addRecord() {
+    struct Record r;
+    FILE *fp = fopen(FILE_NAME, "a");
```
what is fp ?
give proper naming to variables

Suruchibharti committed:
Update CRUD_Operations.c

Updated variable names to use meaningful and consistent camelCase naming convention.
